arraycontext/transform_metadata.py

```diff
@@ -53,6 +53,24 @@ class ElementwiseMapKernelTag(Tag):
         in a finite-element discretization.
     """
 
+
+class ParameterValue(UniqueTag):
+    """A tag that applies to :class:`loopy.ValueArg`. Instances of this tag
+    are initialized with the value of the parameter and may be later used
+    to fix value of the parameter.
+    """
+
+    def __init__(self, value):
+        self.value = value
```

Comment:
Please motivate this better.

Reply:
Expanded description of the purpose of this tag.